### ISSUE TITLE
API download formats

### ISSUE THREAD

**Issue by JFQueralt** [TheIOFoundation/ProjectLockdown#9]
**Problem**

**Objectives**

**Extra information**

**Pending**

**Resources**

**Related Issues**

**Comment by maingockien01** [comment]
@jeff-knurek 
I think we should provide different response content type for a number of selected API endpoints?
I think CSV is really efficient with only 2D table data. For some endpoint like Environment, it's not really efficient to use CSV. Turn Environment response object to CSV is kind of awkward. 
My idea is we will select a list of available content type for each endpoints. JSON will be default available for all endpoint. If users want different format, they need to add it in request header.

**Comment by jeff-knurek** [comment]
good point. @JFQueralt what do you think? it seems like endpoints like `/environment` are just for MAP, and wouldn't ever need to be anything but json. maybe it's only in some variation of the reports (#162) that we need different response formats?

**Comment by jeff-knurek** [comment]
@maingockien01 after quick discussion with @JFQueralt there is just a few endpoints that will use CSV format as output:
	* DSL List
	* Territory List
	* Datapoints List

Also: the other 2 format types (XLS, HXL) aren't required for these endpoints either
